rom/dev/src/start.S

```diff
@@ -108,6 +108,33 @@ _start:
     // Cold Boot
     //
 
+    li t0, 0x30020000  // MBOX_LOCK
+    lw t1, 0(t0)
+    bne t1, x0, mbox_already_locked
+
+    // Zero MBOX
+    la a0, MBOX_ORG         // dest
+    la a1, MBOX_SIZE        // len
+    call _zero_mem256
+
+    // Force unlock mailbox
+    li t0, 0x30020020  // MBOX_UNLOCK
+    li t1, 1
+    sw t1, 0(t0)
+
+    // Tell the SoC it can use the mailbox, as long as we're not doing CSR
+    // generation
+    li t0, 0x300300bc  // CPTRA_DBG_MANUF_SERVICE_REG
+    lw t1, 0(t0)
+
+    andi t1, t1, 0x1   // MFG_FLAG_GEN_IDEV_CSR
+    bne t1, x0, 1f
```

Comment:
Is there a specific reason to move this here instead of later once we've jumped to Rust code?

Reply:
Two reasons:

* We want to clear the mailbox before we clear the ICCM and DCCM so the SoC can stash their measurement as soon as we're done clearing the mailbox. (and we can't clear DCCM from Rust code, as DCCM contains the stack)

* We want to use the _zero_mem256 function, which is much faster than memset.

Reply:
I just meant the ready_for_fw part. It seems like the reason is just to allow the SoC to start doing mailbox writes sooner.

Reply:
Yes, every nanosecond counts when you're trying to meet PCIe reset requirements. Before this PR, they could write to the mailbox immediately after writing fuses. Now they HAVE to wait for ready_for_fw, so let's signal that as soon as we can, rather than forcing them to wait for us to clear iccm and DCCM.